crates/ra_hir_ty/src/tests/traits.rs

```diff
@@ -1991,6 +1991,29 @@ fn test() {
     );
 }
 
+#[test]
+fn fn_item_fn_trait() {
+    check_types(
+        r#"
+//- /main.rs
```

Comment:
For sing-file fixtures, you now can omit the name of the file.

In general, we’ve got rid of single-file/multi-file fixture distinction, it’s now just fixture.